README.md

```diff
@@ -1,2 +1,38 @@
-# riscv-brs-tests
-RISC-V Boot and Runtime Services Test Suite
+# RISC-V Boot and Runtime Services Test Suite.
+
+For more information on The Boot and Runtime Services(BRS) please see：[RISC-V Boot and Runtime Services (BRS) Specification](https://github.com/riscv-non-isa/riscv-brs).
+
+This project is rebuilt based on [rv-brs-test-suite](https://github.com/intel/rv-brs-test-suite) to simplify the brs-tests test environment construction process. Currently includes [edk2-test](https://github.com/tianocore/edk2-test) and [SBI-test](https://gitlab.com/jones-drew/kvm-unit-tests.git).
```

Comment:
Wondering whether we still need a reference to Intel github. Also, as per my knowledge, Drew has upstreamed kvm-unit-tests. If so it is better to remove referring Drew's private fork and provide the public link itself.

Reply:
Can cancel the reference to the Intel project, as the project has been refactored